pkg/boot/linux.go

```diff
@@ -65,10 +64,10 @@ func (li *LinuxImage) Label() string {
 			fmt.Sprintf("initrd=%s", stringer(li.Initrd)),
 		)
 	}
-	if li.DeviceTree != nil {
+	if li.KexecOptions && li.KexecOptions.DTB != nil {
```

Comment:
This is not even legal Go :-) it's C. I'm going to close this PR until such time as it's at least legal Go, sorry :-)